front/src/pages/nft/SingleNFTPageStyles.ts

```diff
@@ -2,6 +2,36 @@ import { createStyles } from "@mantine/core";
 
 export const useStyles = createStyles((theme) => ({
     grid: {
-        padding: [`0 ${theme.spacing.xl}px 0 ${theme.spacing.xl}px`]
+        padding: [`0 ${theme.spacing.xl}px 0 ${theme.spacing.xl}px`],
```

Comment:
what is sense of using this shorthand? you could easily replace it with 0 ${theme.spacing.xl}px - it just describes that top and bottom are 0 and leaf-right are xl